src/main/java/org/asciidoctor/asciidoclet/AsciidoctorRenderer.java

```diff
@@ -127,7 +127,28 @@ private void renderTag(Tag tag, StringBuilder buffer) {
         //print out directly
         buffer.append(tag.name());
         buffer.append(" ");
-        buffer.append(render(tag.text(), true));
+        if ("@param".equals(tag.name())) {
+            // Special handling for @param <T> tags
+            // See http://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#@param
+            String parameterName = getParameterName(tag.text());
+            buffer.append(parameterName).append(' ');
```

Comment:
If you take my patch below, you won't need to manage this space because it will automatically be saved. Though, after removing the type parameter, you should check that the resulting string is not empty because rendering it would be pointless.